static/app/views/replays/replays.tsx

```diff
@@ -44,11 +45,35 @@ function Replays({location}: Props) {
   }, [location]);
 
   const {pathname, query} = location;
-  const {replays, pageLinks, isFetching} = useReplayList({
+  const {replays, pageLinks, isFetching, fetchError} = useReplayList({
     organization,
     eventView,
   });
 
+  if (fetchError && !isFetching) {
+    const reasons = [
+      t('The search parameters you selected are invalid in some way'),
+      t('There is an internal systems error or active issue'),
+    ];
+
+    return (
+      <DetailedError
+        hideSupportLinks
+        heading={t('Sorry, the list of replays could not be found.')}
+        message={
+          <div>
+            <p>{t('This could be due to a handful of reasons:')}</p>
+            <ol className="detailed-error-list">
+              {reasons.map((reason, i) => (
+                <li key={i}>{reason}</li>
+              ))}
+            </ol>
+          </div>
+        }
+      />
+    );
+  }
+
```

Comment:
I noticed we didn't have any error state before, and of course triggered some errors.

Now we show something instead of the empty list component:
<img width="1135" alt="Screen Shot 2022-08-17 at 3 15 34 PM" src="https://user-images.githubusercontent.com/187460/185254057-5223ab58-bc9d-471b-b988-e0c4e8da2702.png">

Reply:
This looks a bit awkward without any Replays chrome, but it's fine for now